INSTALL.rst

```diff
@@ -8,10 +8,10 @@ Building cvc5
         # use --name=<PATH> for custom build directory
         # use --auto-download to download and build missing, required or
         #   enabled, dependencies
-    cd <build_dir>   # default is ./build
-    make             # use -jN for parallel build with N threads
-    make check       # to run default set of tests
-    make install     # to install into the prefix specified above
+    cd <build_dir>            # default is ./build
+    cmake --build .           # use -jN for parallel build with N threads
+    cmake --build . -t check  # to run default set of tests
+    cmake --install .         # to install into the prefix specified above
```

Comment:
This is technically not the same this as `make install` (that would be `cmake --build . -t install`), because it doesn't invoke the build tool at all (CMake copies the files itself), but this shouldn't matter for users.

Reply:
The `--install` and `-t` (the short form of `--target`) options only got added in CMake 3.15, but this shouldn't matter, since we require 3.16 anyway (https://github.com/cvc5/cvc5/blob/main/CMakeLists.txt#L16).

Reply:
The use of `make` instead of `cmake` is intentional, and we prefer to keep it unchanged.